README.md

```diff
@@ -20,15 +20,79 @@ load_balancing_strategy: random
 
 A chat interface using open source models, eg OpenAssistant or Llama. It is a SvelteKit app and it powers the [HuggingChat app on hf.co/chat](https://huggingface.co/chat).
 
-0. [No Setup Deploy](#no-setup-deploy)
-1. [Setup](#setup)
-2. [Launch](#launch)
-3. [Web Search](#web-search)
-4. [Text Embedding Models](#text-embedding-models)
-5. [Extra parameters](#extra-parameters)
-6. [Common issues](#common-issues)
-7. [Deploying to a HF Space](#deploying-to-a-hf-space)
-8. [Building](#building)
+0. [Quickstart](#quickstart)
+1. [No Setup Deploy](#no-setup-deploy)
+2. [Setup](#setup)
+3. [Launch](#launch)
+4. [Web Search](#web-search)
+5. [Text Embedding Models](#text-embedding-models)
+6. [Extra parameters](#extra-parameters)
+7. [Common issues](#common-issues)
+8. [Deploying to a HF Space](#deploying-to-a-hf-space)
+9. [Building](#building)
+
+## Quickstart
```

Comment:
maybe add a link to llama.cpp server docs somewhere. (it's quite nice)

Reply:
I have link here to hf docs https://github.com/huggingface/chat-ui/blob/50edca76928a1fc6d3b131f07ae4315165d18b7f/README.md#L36 and this hf docs contain link to llama server readme https://github.com/huggingface/chat-ui/blob/50edca76928a1fc6d3b131f07ae4315165d18b7f/docs/source/configuration/models/providers/llamacpp.md#L21-L22